src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,23 @@
+@mixin on-phone {
+  @media (min-width: 488px) {
+    .container {
+      --colum-count: 2;
+    }
+  }
+}
+
+@mixin on-tablet {
+  @media (min-width: 768px) {
+    .container {
+      --colum-count: 3;
+    }
+  }
+}
+
+@mixin on-pc {
+  @media (min-width: 1024px) {
+    .container {
+      --colum-count: 4;
+    }
+  }
```

Comment:
we usually use mixins to reuse code somewhere, but these mixins only change the column-count var, I cannot call them reusable